src/topology/algebra/ordered.lean

```diff
@@ -1201,6 +1201,46 @@ begin
     exact ⟨l, la, subset.trans Ioc_subset_Icc_self as⟩ }
 end
 
+section functions
+variables [topological_space β] [linear_order β] [order_topology β]
+
+/-- If `f : α → β` is strictly monotone and surjective, it is everywhere right-continuous. -/
```

Comment:
Please mention `continuous_of_strict_mono_surjective` in the docstring. Do they have the same assumptions?